dcplib/etl/__init__.py

```diff
@@ -178,6 +179,11 @@ def extract(self, query: dict = None, transformer: callable = None, loader: call
     def dispatch_callbacks(self, bundle_uuid, bundle_version, transformer, loader):
         bundle_path = f"{self.sd}/bundles/{bundle_uuid}.{bundle_version}"
         bundle_manifest_path = f"{self.sd}/bundle_manifests/{bundle_uuid}.{bundle_version}.json"
+        if not os.path.exists(bundle_path):
```

Comment:
This would occur if ES is out of sync with the DSS? ex. a bundle is deleted from the DSS but still exists in ES

Reply:
No. I don't know why it would occur. It probably occurs because of test bundles in dev that don't have valid metadata. But when we do get such a bundle, we currently crash because nothing is pulled down from it and no bundle directory exists on the local filesystem.